### PR TITLE
Fix: replace broken private Slack link with GitHub issue link in repo…

### LINKED ISSUE #7750: Bug: "Report an issue" link in simulator sends anonymous users to a Slack sign-in wall instead of the channel
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Describe the bug

The simulator's built-in "Report an issue" panel (flag icon in the bottom toolbar) links out to https://circuitverse-team.slack.com/archives/C010HK6TWQG. Since circuitverse-team.slack.com is a private Slack workspace, this URL only resolves to the channel for users already signed into that workspace. For anonymous visitors and anyone not already a member (i.e. almost every user trying to report a bug for the first time), the link redirects to a generic Slack sign-in/landing page instead of the channel , so there is no way to actually report the issue from here. This makes the platform's only in app bug reporting path non functional for the vast majority of users.

### Steps To Reproduce

1-Go to circuitverse.org/simulator (while logged out, or in an incognito window) 
2-Click the "Report an issue" (flag) icon in the bottom toolbar 
3-Click "Click Here to report an issue" in the panel that opens
4-See error: you land on a generic Slack sign-in/redirect page, not the intended channel

### Expected Behavior

The link should either point to a public Slack invite link (so new users can join and then see the channel), or route to a public bug-report form / GitHub issue template instead of a private Slack archive URL, since Slack archive links are not accessible to non-members.

### Screenshots

Image

Image

### Device Information [optional]

```markdown
- OS: macOS
- Browser:Brave
- version:latest
```

### Additional context

_No response_

### Are you working on this issue?

Yes

### PR DESCRIPTION
**Pull request by harsh-mishra123**
Fixes #7750

#### Describe the changes you have made in this PR -
The simulator's "Report an issue" panel linked to a private Slack workspace archive URL (`circuitverse-team.slack.com/archives/C010HK6TWQG`). This URL only resolves for users already signed into that private workspace — for anonymous visitors and non-members (i.e. almost everyone trying to report a bug for the first time), it redirected to a Slack sign-in wall instead, making the platform's only in-app bug-reporting path non-functional.

I also checked the Slack invite link used elsewhere in the codebase (`app/views/layouts/mailer.html.erb`) as a possible source for a working link, but confirmed that invite link has also expired (screenshot in the comment on #7750). Since I don't have Slack admin access to generate a fresh invite, I replaced the broken link with a direct link to GitHub's issue creation flow (`github.com/CircuitVerse/CircuitVerse/issues/new/choose`) instead — it doesn't expire and takes users straight to where bug reports are actually triaged.

---
## Code Understanding and AI Usage
**Did you use AI generated code (ChatGPT, Claude, Copilot, etc.) in any part of this PR?**
- [x] No, I wrote all the code myself

**If you used AI assistance:**
- [x] I have reviewed every single line of the AI-generated code
- [x] I can explain the purpose and logic of each function/component I added
- [x] I have tested edge cases and understand how the code handles them
- [x] I have modified the AI output to follow this project's coding standards and conventions

**Explain your implementation approach:**
The problem: an ERB view (`app/views/simulator/edit.html.erb`) contained a hardcoded link to a private Slack channel archive, which fails for any user not already logged into that specific Slack workspace — effectively breaking the entire bug-report entry point in the simulator UI.

I considered two alternatives:
1. Point to the Slack invite link already used in the app's mailer template — rejected after testing it and finding it had also expired.
2. Point to a fresh Slack invite link — not possible, since generating one requires Slack workspace admin access I don't have.

I chose to link directly to GitHub's "new issue" chooser instead, since it's a URL I don't need any special access to generate, it never expires, and it sends users to the exact place bug reports are supposed to end up anyway (this very tracker).

The change itself is a single-line HTML/ERB edit: the `<a href="...">` link's `href` and surrounding sentence were updated, and I added `target="_blank" rel="noopener noreferrer"` so the link opens in a new tab (standard practice for external links, and it means a user reporting a bug doesn't lose their in-progress circuit in the simulator tab).

I verified the fix by running the app locally via Docker, opening `/simulator`, clicking the report-issue flag icon, and confirming the panel shows the updated text and link, and that clicking it opens the correct GitHub page in a new tab. I also grepped the codebase afterward to confirm no remaining references to the old Slack archive URL.

---
## Checklist before requesting a review
- [x] I have added proper PR title and linked to the issue
- [x] I have performed a self-review of my code
- [x] I can explain the purpose of every function, class, and logic block I added
- [x] I understand why my changes work and have tested them thoroughly
- [x] I have considered potential edge cases and how my code handles them
- [ ] If it is a core feature, I have added thorough tests
- [x] My code follows the project's style guidelines and conventions

---
Note: Please check **Allow edits from maintainers** if you would like us to assist in the PR.

## Summary by CodeRabbit

* **Updates**
  * Issue-reporting guidance now directs users to create issues in the CircuitVerse GitHub repository instead of using Slack.
  * The repository link opens securely in a new browser tab.

### CODE REVIEW

File: app/views/simulator/edit.html.erb [hunk 1]
```diff
@@ -751,11 +751,11 @@
...
 
         <%# TEMPORARY %>
-        <center>
-        Bug reporting from here will be back soon, until then help make CircuitVerse better by reporting simulator issues to us on Slack.
-        <br><br><a href="https://circuitverse-team.slack.com/archives/C010HK6TWQG" style="color: blue;">Click Here to report an issue</a><br><br>
-        Thank you for your time.
-        </center>
+       <center>
+Help make CircuitVerse better by reporting simulator issues on our GitHub repository.
+<br><br><a href="https://github.com/CircuitVerse/CircuitVerse/issues/new/choose" style="color: blue;" target="_blank" rel="noopener noreferrer">Click Here to report an issue</a><br><br>
```
[the remark this answers is not in the document]
Comment (harsh-mishra123) [reply app/views/simulator/edit.html.erb:756]: @

> _🎯 Functional Correctness_ | _🟠 Major_ | _🏗️ Heavy lift_
> 
> 🧩 Analysis chain
> 🌐 Web query:
> 
> `GitHub creating issues signed out unauthenticated issue forms private public repositories docs`
> 
> 💡 Result:
> 
> GitHub does not support unauthenticated users creating issues directly on GitHub repositories [1][2]. All users must be logged into a valid GitHub account to create an issue, regardless of whether the repository is public or private [2][3]. While GitHub provides "Issue Forms" (a structured way to collect information for issues using YAML templates), these forms are still subject to the same authentication requirements as standard issues [4][5]. When a user interacts with an issue form, they must be authenticated with GitHub to submit it [2]. If you wish to allow unauthenticated or anonymous users to submit feedback or bug reports that end up as issues in your repository, you must implement a workaround using an intermediary application [6][7][8]: 1. Create a custom web form or interface hosted on your own infrastructure [6]. 2. When a user submits this form, your application acts as an intermediary, using an authenticated GitHub API token (associated with a specific account you control) to create the issue in your repository on their behalf [6][7]. 3. This process effectively offloads the authentication requirement to your application rather than requiring the end user to have a GitHub account [6]. Note that repository administrators can further restrict issue creation to "collaborators only" (users with specific repository permissions), which prevents even authenticated members of the public from opening issues [9][10]. This setting can be managed in the repository's settings under Features > Issues [9][10].
> 
> Citations:
> 
> * 1: https://docs.github.com/en/rest/issues/issues
> * 2: https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/creating-an-issue
> * 3: https://github.com/github/docs/blob/main/content/issues/tracking-your-work-with-issues/using-issues/creating-an-issue.md
> * 4: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
> * 5: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository
> * 6: https://github.com/bliepp/IssueForms
> * 7: https://stackoverflow.com/questions/31546332/submitting-issues-to-a-public-github-project-anonymously
> * 8: https://www.reddit.com/r/webdev/comments/1rva6h/git_reports_allow_anonymous_users_to_submit_bug/
> * 9: https://github.blog/changelog/2026-06-29-restrict-issue-creation-to-collaborators-only/
> * 10: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/disabling-issues
> 
> 🌐 Web query:
> 
> `Issue `#7750` CircuitVerse anonymous users report simulator issues`
> 
> 💡 Result:
> 
> There is no publicly indexed GitHub issue labeled `#7750` in the CircuitVerse repositories as of August 7, 2026. While CircuitVerse has experienced various simulator-related issues [1][2][3], no specific entry `#7750` is documented regarding reports from anonymous users [4][5]. CircuitVerse has faced historical challenges with its "Report an Issue" system, including concerns about persistent abuse by anonymous users [6], which may have led to adjustments in how reports are handled or tracked. Recent simulator issues have been actively tracked in the cv-frontend-vue and CircuitVerse/CircuitVerse repositories, focusing on authentication flows [7][8], project loading [1], and UI regressions [2][9], rather than a specific issue `#7750` [4].
> 
> Citations:
> 
> * 1: [Project wont load #7261](https://github.com/CircuitVerse/CircuitVerse/issues/7261)
> * 2: [TESTBENCH panel overlaps Circuit Elements icons/labels when a category is expanded #7690](https://github.com/CircuitVerse/CircuitVerse/issues/7690)
> * 3: [JK flip-flop ignoring timing 🐞 Bug #7256](https://github.com/CircuitVerse/CircuitVerse/issues/7256)
> * 4: https://github.com/CircuitVerse/CircuitVerse/issues
> * 5: https://github.com/CircuitVerse/CircuitVerse
> * 6: [Simulator "Report An Issue" System  persistent abuse. #4436](https://github.com/CircuitVerse/CircuitVerse/issues/4436)
> * 7: [🐞 Bug: [vuesim] auth webview model isn't compatible for omniauth cv-frontend-vue#1110](https://github.com/CircuitVerse/cv-frontend-vue/issues/1110)
> * 8: [Fix: [vuesim] Redirect user to rails devise login form for auth in vuesim cv-frontend-vue#1114](https://github.com/CircuitVerse/cv-frontend-vue/pull/1114)
> * 9: [🐞 Bug: (vuesim) Circuit preview not shown in the dashboard of projects #7516](https://github.com/CircuitVerse/CircuitVerse/issues/7516)
> 
> 🏁 Script executed:
> 
> ```shell
> #!/bin/bash
> set -eu
> 
> echo "Files matching simulator/edit.html.erb:"
> fd -a 'edit.html.erb' . | sed 's#^\./##'
> 
> echo
> echo "Relevant lines in app/views/simulator/edit.html.erb:"
> if [ -f app/views/simulator/edit.html.erb ]; then
>   nl -ba app/views/simulator/edit.html.erb | sed -n '740,770p'
> else
>   echo "app/views/simulator/edit.html.erb not found"
> fi
> 
> echo
> echo "Repository references to issue `#7750`:"
> rg -n --fixed-strings "`#7750`" . || true
> 
> echo
> echo "Repository references to anonymous issue reporting / simulator issue reporting:"
> rg -n -i 'anonymous|not authenticated|report simulator issue|report an issue|GitHub repository' app || true
> ```
> 
> Repository: CircuitVerse/CircuitVerse
> 
> Length of output: 1630
> 
> 🏁 Script executed:
> 
> ```shell
> #!/bin/bash
> set -eu
> 
> echo "Relevant lines in app/views/simulator/edit.html.erb:"
> awk '{printf "%7d  %s\n", NR, $0}' app/views/simulator/edit.html.erb | sed -n '740,770p'
> 
> echo
> echo "Inspect any existing public report form/controller references:"
> rg -n -i 'report|reports|feedback|anonymous|unauthenticated|bug|issue' app/models app/controllers app/views -g '*.rb' -g '*.erb' || true
> ```
> 
> Repository: CircuitVerse/CircuitVerse
> 
> Length of output: 6566
> 
> **Keep an anonymous reporting path.**
> 
> GitHub issue submission requires authentication, so the `issues/new/choose` link does not preserve the anonymous reporting path. Add back or restore an unauthenticated local report form/API path, or confirm that requiring GitHub sign-in is an accepted requirement change.


Thanks for the automated feedback. However, implementing a custom intermediary application with API token proxying is out of scope for issue #7750. 
The objective of this PR is specifically to replace the dead/broken private Slack link with CircuitVerse's public GitHub issue chooser link (`/issues/new/choose`), which is standard practice across open-source projects. 
If maintainers wish to support anonymous in-app issue submission in the future, that should be designed and tracked under a separate feature request.
